docs/UpstreamRelation.md

```diff
@@ -0,0 +1,45 @@
+# Upstream Relation
+
+This document describes what is purpose of this repository,
+why do we provide another OpenTelemetry Collector distrubution and
+how look like our versioning and breaking changes policies.
+
+## Purpose of Sumo Logic OT Distro
+
+Sumo Logic OT Distro is the OpenTelemetry Collector with addition of Sumo Logic specific changes.
+
+**Our aim is to extend and not to replace the OpenTelemetry Collector.**
+
+We work close with the OpenTelemetry community in order to improve overall experience by:
```

Comment:
```suggestion
We work closly with the OpenTelemetry community in order to improve overall experience by:
```
but I'm not english expert ;)

Reply:
That was my first version, so it should be good :+1: